homeassistant/setup.py

```diff
@@ -18,6 +18,26 @@
 
 ATTR_COMPONENT = "component"
 
+BASE_PLATFORMS = {
+    "alarm_control_panel",
```

Comment:
`air_quality` is missing and `notify` too even though it isn't an entity component.

Reply:
It's a bit annoying that we need to remember to extend this list when we add a new base entity integration. Could we extend the manifest for this purpose? @balloob?

Reply:
> air_quality is missing and notify too even though it isn't an entity component.

Good catch. Will add them.